integration/helpers/base_test.py

```diff
@@ -115,9 +116,17 @@ def setUp(self):
         self.output_file_path = None
         self.sub_input_file_path = None
 
+    @retry(
+        stop=stop_after_attempt(3),
```

Comment:
```suggestion
        stop=stop_after_attempt(5),
```

Reply:
I pushed the same change at the same time lol